src/gui/properties/propertieswidget.cpp

```diff
@@ -382,127 +395,119 @@ void PropertiesWidget::saveSettings()
 
-        m_ui->label_completed_on_val->setText(m_torrent->completedTime().isValid() ? m_torrent->completedTime().toString(Qt::DefaultLocaleShortDate) : "");
+            m_ui->labelCompletedOnVal->setText(m_torrent->completedTime().isValid() ? m_torrent->completedTime().toString(Qt::DefaultLocaleShortDate) : "");
 
-        m_ui->label_added_on_val->setText(m_torrent->addedTime().toString(Qt::DefaultLocaleShortDate));
+            m_ui->labelAddedOnVal->setText(m_torrent->addedTime().toString(Qt::DefaultLocaleShortDate));
 
-        if (m_torrent->hasMetadata()) {
-            m_ui->label_total_pieces_val->setText(tr("%1 x %2 (have %3)", "(torrent pieces) eg 152 x 4MB (have 25)").arg(m_torrent->piecesCount()).arg(Utils::Misc::friendlyUnit(m_torrent->pieceLength())).arg(m_torrent->piecesHave()));
+            if (m_torrent->hasMetadata()) {
+                m_ui->labelTotalPiecesVal->setText(tr("%1 x %2 (have %3)", "(torrent pieces) eg 152 x 4MB (have 25)").arg(m_torrent->piecesCount()).arg(Utils::Misc::friendlyUnit(m_torrent->pieceLength())).arg(m_torrent->piecesHave()));
+
+                if (!m_torrent->isSeed() && !m_torrent->isPaused() && !m_torrent->isQueued() && !m_torrent->isChecking()) {
+                    // Pieces availability
+                    showPiecesAvailability(true);
+                    m_piecesAvailability->setAvailability(m_torrent->pieceAvailability());
+                    m_ui->labelAverageAvailabilityVal->setText(Utils::String::fromDouble(m_torrent->distributedCopies(), 3));
+                }
+                else {
+                    showPiecesAvailability(false);
+                }
 
-            if (!m_torrent->isSeed() && !m_torrent->isPaused() && !m_torrent->isQueued() && !m_torrent->isChecking()) {
-                // Pieces availability
-                showPiecesAvailability(true);
-                pieces_availability->setAvailability(m_torrent->pieceAvailability());
-                m_ui->avail_average_lbl->setText(Utils::String::fromDouble(m_torrent->distributedCopies(), 3));
+                // Progress
+                qreal progress = m_torrent->progress() * 100.;
+                m_ui->labelProgressVal->setText(Utils::String::fromDouble(progress, 1) + "%");
+                m_downloadedPieces->setProgress(m_torrent->pieces(), m_torrent->downloadingPieces());
             }
             else {
                 showPiecesAvailability(false);
             }
-
-            // Progress
-            qreal progress = m_torrent->progress() * 100.;
-            m_ui->progress_lbl->setText(Utils::String::fromDouble(progress, 1) + "%");
-            downloaded_pieces->setProgress(m_torrent->pieces(), m_torrent->downloadingPieces());
         }
-        else {
-            showPiecesAvailability(false);
-        }
-
         break;
-    }
-
-    case PropTabBar::TRACKERS_TAB: {
```

Comment:
I would prefer with the curly brackets.

Reply:
IMO, if some `case` require curly braces (i.e. it needs some local variables) it's good point to extract the code into function.

Reply:
What's the point of the guidelines then.